lib/jnpr/junos/device.py

```diff
@@ -1008,6 +1020,9 @@ def open(self, *vargs, **kvargs):
                 ssh_config=self._sshconf_lkup(),
                 device_params={'name': 'junos', 'local': False})
 
+            self._conn._session.transport.set_keepalive(
```

Comment:
line 910 we are setting it to None for on-box if condition. So on on-box are not reaching line 924, hence converting None to int will throw exception.